state_transfer/endpoint/route/route.go

```diff
@@ -29,30 +31,27 @@ type RouteEndpoint struct {
 	namespacedName types.NamespacedName
 }
 
-func NewEndpoint(namespacedName types.NamespacedName, port int32, eType RouteEndpointType, labels map[string]string) endpoint.Endpoint {
+func NewEndpoint(namespacedName types.NamespacedName, eType RouteEndpointType, labels map[string]string) endpoint.Endpoint {
 	if eType != EndpointTypePassthrough && eType != EndpointTypeInsecureEdge {
 		panic("unsupported endpoint type for routes")
 	}
 	return &RouteEndpoint{
 		namespacedName: namespacedName,
 		labels:         labels,
 		endpointType:   eType,
-		port:           port,
 	}
 }
 
 func (r *RouteEndpoint) Create(c client.Client) error {
-	err := r.createRouteService(c)
-	if err != nil {
-		return err
-	}
+	errs := []error{}
 
-	err = r.createRoute(c)
-	if err != nil {
```

Comment:
can we keep the error message and log the error just to give us a changedebugging?

If the changes are non-trivial, let's file an issue